### ISSUE TITLE
Model training code

### ISSUE THREAD

**Issue by mikkelam** [faezetta/VMMRdb#1]
Hi! 

Awesome research. I was wondering if you could release the code for training the model?

**Comment by keishinkickback** [comment]
Hi @mikkelam,

Thanks! I am one of the co-authors of this paper. We used [this torch](https://github.com/facebook/fb.resnet.torch) code to train our models. Also, you can train your own models with different deep-learning frameworks by treating different folders in our dataset as different classes. Please let me know if you have any further questions :)

cc: @faezetta

**Comment by mikkelam** [comment]
Thank you for your reply. I will most likely try and gather the models in their respective production years to avoid having to look at accuracy@5 and compare my results with you guys while using pytorch
